tests/integration/conftest.py

```diff
@@ -662,3 +700,143 @@ def kernel(shell):
 
         # Reboot after clean-up
         shell("tmt-reboot -t 600")
+
+
+@pytest.fixture()
+@pytest.mark.yum_conf_exclude
+def yum_conf_exclude(shell, backup_directory, request):
```

Comment:
The request parameter is not used anywhere

Reply:
fixed